### DISCUSSION TITLE
Increasing the width of the console

### DISCUSSION THREAD

**Discussion by VacAge** [prplwtf/fetch-nebula#91] [Support] [answered]
Good afternoon, I'm trying to edit the theme style a little and I can't figure out how to expand the console the way it looks very narrow against a wide screen....

**Comment by VacAge** [comment]
There is too much unallocated space, which would be nice to expand for the console. How could I do this?
![image](https://github.com/user-attachments/assets/40f51122-cd92-4e56-8265-b794581dab70)
And how to do the same with the file editor...

**Comment by prplwtf** [comment, marked as answer]
I recommend taking a look at Nebula's floating window styles and styling your page based on that. Floating windows, compared to normal Nebula, do not have a width limit.

**Reply by NoExtraSauce** [reply]
Where would i find this option for floating windows?

**Reply by VacAge** [reply]
++

**Reply by NoExtraSauce** [reply]
> ++

I figured it out, you need to right click the side panel buttons. 

But it would still be nice if console and files page were the full length. even if its a customize option and off by default

![image](https://github.com/user-attachments/assets/bff6bba9-5c66-45f7-8895-064306335953)
